app/components/calls_to_action/adviser_block_component.rb

```diff
@@ -0,0 +1,22 @@
+module CallsToAction
+  class AdviserBlockComponent < ViewComponent::Base
+    attr_reader :title, :image, :title_color, :text, :link_text, :link_target
+
+    def initialize(
+      title: "Get an adviser",
+      image: "adviser-block-promo.jpg",
+      title_color: "yellow",
+      text: "An adviser with years of teaching experience can help you become a teacher. Chat by phone, text, or email as little or often as you need.",
```

Comment:
```suggestion
      text: "An adviser with years of teaching experience can help you become a teacher.",
```
Could take out the second sentence to make it shorter/more concise?